src/components/renders/bootstrap/select/select-render.tsx

```diff
@@ -79,16 +79,28 @@ export class SelectRender implements Renderer {
         }
       };
 
-      return [
-        <gx-bootstrap />,
-        <select {...attris}>
-          {this.options.map(({ disabled, innerText, selected, value }) => (
-            <option disabled={disabled} selected={selected} value={value}>
-              {innerText}
-            </option>
-          ))}
-        </select>
-      ];
+      return this.component.suggest
+        ? [
+            <input list={this.selectId}></input>,
+
+            <datalist id={this.selectId}>
+              {this.options.map(({ disabled, innerText, selected, value }) => (
+                <option disabled={disabled} selected={selected} value={value}>
+                  {innerText}
+                </option>
+              ))}
+            </datalist>
```

Comment:
We should also include `gx-bootstrap` (at least until we use the new design). We should also set the `form-input` class to the `<input>` element.

Reply:
Bootstrap...
Ok ok, I will add it! 👌

Reply:
Bootstrap tag added in commit f25ad5a
("I forgot to add a description for this change in the last commit 😅")